jaeger_client/sampler.py

```diff
@@ -192,6 +195,13 @@ def __eq__(self, other):
         d1['last_tick'] = d2['last_tick']
         return d1 == d2
 
+    def update(self, max_traces_per_second):
```

Comment:
i dun gon goofed, still the update logic doesn't check for equality so I'll make that change